browser-extension/mv3/src/page-scripts/ajaxRequestInterceptor/index.js

```diff
@@ -0,0 +1,16 @@
+import { initFetchInterceptor } from "./fetch";
+import { initXhrInterceptor } from "./xhr";
+
+export const initAjaxRequestInterceptor = () => {
```

Comment:
No need to export it
<img width="830" alt="image" src="https://github.com/requestly/requestly/assets/56021937/337bba10-b9d0-417b-846d-98e6e77973da">